api/tests/core/users/test_generator.py

```diff
@@ -9,9 +10,42 @@
 
 
 class UserGeneratorTest:
+    def assert_user_at_email_validation(self, user: users_models.User):
```

Comment:
Cool ces petits test, très clairs à lire !


<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**2 🎯 suggestions:** ‏

- Ajouter les tests sur les créations de fraud checks (vérifier qu'ils existent)
- Le nommage me perturbe un peu. Quand je lis `assert_user_at_phone_validation` je m'attend à ce que la step "phone validation" ne soit pas encore faite. 
  - on peut renommer `user_passed_phone_val` ou quelque chose dans le genre